packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.6.0-dev (Mar 17, 2022)
+## v1.5.0 (Mar 17, 2022)
+## v1.5.0-alpha.1 (Mar 17, 2022)
+## v1.5.0-alpha.0 (Mar 17, 2022)
 ## v1.5.0-dev (Jan 28, 2022)
```

Comment:
We might want to do the same for pwa.

Reply:
Thanks @alexvuong, the Changelogs are now updated with the most relevant PRs.